docs/user/debugging/android.md

```diff
@@ -54,6 +54,12 @@ persist until the application is closed or manually reset.
 >   --es tagPings test-metrics-ping
 > ```
 
+Debugging features in Android can also be enabled using environment variables.
+For more information on the available features accessible through this method
+and how to enable them, see [Enabling debugging features through environment variables](./index.md).
+
+These environment variables must be set on the device that is running the application.
```

Comment:
Should we add a note here saying we actually don't know how to do this? Or should we just add not add this to the Android docs yet?

Reply:
Let's remove these docs, if we don't support it. Or clearly state that debugging using environment variables is not supported on Android.

Reply:
Like I said in the other comment it is supported. My thinking when I added this docs was to add it and let the user try to figure out how to set environment variables in an Android device. When I saw it in the PR I started doubting this and thought maybe we could just remove this docs from Android for now, with the intention of not bringing attention to something we might not know how to answer in case a user asks us.

Reply:
I don't think "it's supported": users can't actually use it, by design. If they ask us how to do this, we'd just shrug 🤷 - Let's clearly state that while the Glean SDK can read environment variables, we don't know how to set them on the device, so other methods should be used.

Reply:
I agree, I will remove this docs and add a note on `index.md`.